spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -120,6 +121,26 @@
         'PresentationChanged' => 'false',
       )
     end
+
+    it 'notes that address line 2 was present' do
+      expect(response.to_h).to include(address_line2_present: true)
+    end
+  end
+
+  context 'when there is no address line 2' do
+    body_no_line2 = JSON.parse(LexisNexisFixtures.true_id_response_success_2).tap do |json|
+      json['Products'].first['ParameterDetails'] = json['Products'].first['ParameterDetails'].
+        select { |f| f['Name'] != 'Fields_AddressLine2' }
+    end.to_json
+    let(:success_response_no_line2) do
+      instance_double(Faraday::Response, status: 200, body: body_no_line2)
+    end
+
+    let(:response) { described_class.new(success_response_no_line2, config) }
+
+    it 'notes that address line 2 was not present' do
+      expect(response.to_h).to include(address_line2_present: false)
+    end
```

Comment:
can we add a spec that checks that `pii_from_doc` includes address2? Or is that for a separate step?

Reply:
Good points! Thanks for pointing to improved tests. Fixed with d0ede824b01c6b51f8d4edd68557193867789478.